python/py-upt-macports/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+set git_hash        20b55dcc6a06f48da8a22c415bc74315dda5cb2e
+github.setup        macport upt-macports ${git_hash}
+version             0.1
+name                py-${github.project}
+revision            0
+
+maintainers         {@korusuke somaiya.edu:karan.sheth} openmaintainer
+description         MacPorts backend for upt.
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/macports/upt-macports/
```

Comment:
you can remove ```homepage```, ```master_sites``` and ```distname```; these are all the defaults already set by the github PG (didn't work for you because of the typo in ```github.setup```.